src/components/breadcrumbs/breadcrumbs.spec.tsx

```diff
@@ -0,0 +1,247 @@
+import { h } from '@stencil/core';
+import { newSpecPage, SpecPage } from '@stencil/core/testing';
+import { BreadcrumbsItem } from '@limetech/lime-elements';
+import { Breadcrumbs } from './breadcrumbs';
+
+let page: SpecPage;
+let handleSelect: jest.Mock;
+const buttonLikeItems: BreadcrumbsItem[] = [
+    {
+        text: 'step 1',
+        icon: {
+            name: 'fish',
+            color: 'rgb(var(--color-red-default))',
+        },
+    },
+    {
+        text: 'Step 2',
+        icon: {
+            name: 'cat',
+            color: 'rgb(var(--color-orange-default))',
+        },
+    },
+    {
+        text: 'Step 3',
+        icon: {
+            name: 'dog',
+            color: 'rgb(var(--color-blue-default))',
+        },
+    },
+];
+
+const hyperlinkItems: BreadcrumbsItem[] = [
+    {
+        text: 'Home',
+        link: {
+            href: '../../../..',
+            title: 'Start',
+        },
+    },
+    {
+        text: 'Products',
+        link: {
+            href: '../../../',
+            title: 'See all of our products',
+        },
+    },
+    {
+        text: 'Phones',
+        link: {
+            href: '../../',
+        },
+    },
+    {
+        text: 'Accessories',
+        link: {
+            href: '../',
+        },
+    },
+    {
+        text: 'Earphones',
+    },
+];
+
+describe('limel-breadcrumbs', () => {
+    describe('with button-like elements', () => {
+        beforeEach(async () => {
+            handleSelect = jest.fn();
+            page = await newSpecPage({
+                components: [Breadcrumbs],
+                template: () => (
+                    <limel-breadcrumbs
+                        items={buttonLikeItems}
+                        onSelect={handleSelect}
+                    />
+                ),
+            });
```

Comment:
We are re-writing these lines of codes multiple time. I think we can have a function like `createComponent(items BreadcrumbsItem[], divider?: string)` and put these lines of codes there

Reply:
thanks for the helps 🙏